java/code/src/com/redhat/rhn/manager/content/ContentSyncManager.java

```diff
@@ -119,9 +119,12 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import javax.persistence.Embeddable;
+
 /**
  * Content synchronization logic.
  */
+@Embeddable
```

Comment:
Huh? Why do we need this?

Reply:
IIRC correctly it should be used because it's used by some entity but there's no table associated to it...but it has sense to try to remove it.

Reply:
indeed, not required. I removed it